mk/config.mk

```diff
@@ -1022,6 +1022,9 @@ CFG_CORE_ASYNC_NOTIF ?= n
 $(call force,_CFG_CORE_ASYNC_NOTIF_DEFAULT_IMPL,$(CFG_CORE_ASYNC_NOTIF))
 endif
 
+# Enable callout service
+CFG_CALLOUT ?= $(CFG_CORE_ASYNC_NOTIF)
```

Comment:
Callout does not depend on needs async notif and may have async notif without callout.
I would rather expect `CFG_CALLOUT ?= n`. But that's not a strong opinion. Disabling callout because of pager constraints can be set from the platform _conf.mk_ file if needed.

Reply:
I'm trying to assign a sensible default value. If you have `CFG_CORE_ASYNC_NOTIF` enabled it's likely that you wouldn't mind `CFG_CALLOUT`. Without `CFG_CORE_ASYNC_NOTIF`, then `CFG_CALLOUT` isn't that useful.

Reply:
Yes, it's still possible for the platform to override the default value. The assignment isn't a dependency.
If we set this to default `n`, then all platforms that want to keep up with new features must update.

Reply:
Ok, fine as-is.